### PR TITLE
Feature/review org setup

### PR DESCRIPTION
**Pull request by anxolin**
Simplifies the config and fix some issues.

Also it setups the dates of locking, claiming and redeeming.

The config info is defined in the following files:

* **dao**: basic data of the DAO (like the name of the ORG)
* **bootstrap**: Info related to the initiation process
* **initalRepDistribution**: Amounts of REP assigned for the different distribution processes within the initiation process
* **genTokenAddress**: GEN address for all networks
* **timePeriods**: Locking, Claiming, and Redeem dates
* **voting**:  Genesis Protocol voting machine config

### CODE REVIEW

**Comment by anxolin** [comment]
The config info is defined in the following files:

* **dao**: basic data of the DAO (like the name of the ORG)
* **bootstrap**: Info related to the initiation process
* **initalRepDistribution**: Amounts of REP assigned for the different distribution processes within the initiation process
* **genTokenAddress**: GEN address for all networks
* **timePeriods**: Locking, Claiming, and Redeem dates
* **voting**:  Genesis Protocol voting machine config

File: migrations/14_SCHEMES_SchemeRegistrar.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
   UPGRADE_CONTROLLER,
   CALL_DELEGATECALL
-} } = registerScheme
+} = SchemePermissions
 
 const getDaostackContract = require('../src/helpers/getDaostackContract')(web3, artifacts)
 
 
-module.exports = async function (deployer) {
+module.exports = async function (deployer, network) {
```
Comment (Velenir) [inline migrations/14_SCHEMES_SchemeRegistrar.js]: Why add `network` if you don't use it?

File: migrations/helpers/schemeUtils.js [hunk 2]
```diff
@@ -32,6 +39,28 @@ async function registerScheme ({
...
+  assert(parameters, `The parameter parameters was not defined`)
+  assert(typeof parameters === 'object', `The parameter should be an array`)
+  
+  console.log('\nSet scheme parameters:')
+  Object.keys(parameters).forEach(paramName => {
+    console.log(`  - ${paramName}: ${parameters[paramName]}`)
+  })
 
-module.exports = registerScheme
+  const paramList = Object.values(parameters)
```
Comment (Velenir) [inline migrations/helpers/schemeUtils.js]: This requires fixed order of `parameters` object keys, so 
```
parameters: {
      voteParams: genesisProtocolParamsHash,
      intVote: genesisProtocolAddress,
      contractToCall: dutchXContractAddress
    }
```
can't be given in any other order, which is usually not what is expected when accepting an object input.
Better make `parameters` in to an array, imho.

**Comment by orenyodfat** [comment]
Please note that the voting machine params will be different from what in this or as we plan to update gp plus it params .
Travis red

**Comment by anxolin** [comment]
The test didn't pass cause is failing in the MGN tests since I update to arc's last version. We'll check that. Anyways, we can approve the other PRs too, and their tests passes :) 

Also, @orenyodfat , can you be more specific with the "voting machine params". We didn't change anything on the voting machine. I understand it'll be reviewed in detail later, but is there a change we should be aware of.

File: migrations/helpers/schemeUtils.js (hunk 2 above)
Comment (anxolin) [reply migrations/helpers/schemeUtils.js]: Very good feedback, thanks Dima. I will send a new commit soon to address this

File: migrations/14_SCHEMES_SchemeRegistrar.js (hunk 1 above)
Comment (anxolin) [reply migrations/14_SCHEMES_SchemeRegistrar.js]: Network is not required, but extrangely it requires deployer, even if you don't use it. I think truffle 5 does a check somehow. 

If you remove the deployer, the migrations fail.

**Comment by anxolin** [comment]
@orenyodfat @Velenir  Please, can you review the PR? 

- The test pass: Thanks to Dima 👍 
- Travis uses now solium linter
- Travis uses now eslint linter
- The lint warn/errors has been resolved
- @Velenir , I applied the change you suggested about using an array for the parameters

**Pull request merged**